src/components/Chart.jsx

```
@@ -1,25 +1,40 @@
-import barData from "../dataset/PieData.json";
+import PropTypes from "prop-types";
+import LineChartComponent from "./LineChart";
 import PieChartProject from "./PieChart";
 import ZoomLineChart from "./ZoomLineChart";
 import ParentLineChart from "./ParentLineChart";
```

ESLint

[error] 'ParentLineChart' is defined but never used.

(no-unused-vars)